Source/WebGPU/WebGPU/Device.h

```diff
@@ -153,6 +165,9 @@ class Device : public WGPUDeviceImpl, public ThreadSafeRefCountedAndCanMakeThrea
     HardwareCapabilities m_capabilities { };
 
     const Ref<Adapter> m_adapter;
+#if defined(__OBJC__) && COREVIDEO_SUPPORTS_METAL
+    CVMetalTextureCacheRef m_coreVideoTextureCache;
```

Comment:
I think this has to be `RetainPtr<CVMetalTextureCacheRef> m_coreVideoTextureCache;` because ARC doesn't `CFRelease()` CF types.